README.md

```diff
@@ -254,32 +254,32 @@ export type Animate = 'lazy';
 | `point.visible`           | boolean       | optional | It is used to indicate no. of slides and also shows the current active slide.                                                                                                                                                 |
 | `point.hideOnSingleSlide` | boolean       | optional | It is used to hide the point indicator when slide is less than one.                                                                                                                                                           |
 | `touch`                   | boolean       | optional | It is used to active touch support to the carousel.                                                                                                                                                                           |
 | `easing`                  | string        | optional | It is used to define the easing style of the carousel. Only define the ease name without any timing like `ease`,`ease-in`                                                                                                     |
 | `loop`                    | boolean       | optional | It is used to loop the `ngu-item ngu-tile`. It must be true for `interval`                                                                                                                                                    |
-| `animation`               | string        | optional | It is used to animate the sliding items. currently it only supports `lazy`. more coming soon and also with custom css animation option                                                                                        |
-| `custom`                  | string        | optional | It is you to define the purpose of the carousel. currently it only supports `banner`. more coming soon and also with custom css animation option                                                                              |
-| `RTL`                     | boolean       | optional | This option enable the `rtl` direction and act as rtl. By default it is `ltr`                                                                                                                                                 |
+| `animation`               | string        | optional | It is used to animate the sliding items. currently it only supports `lazy`. more coming soon and also with custom CSS animation option                                                                                        |
+| `custom`                  | string        | optional | It is you to define the purpose of the carousel. Currently, it only supports `banner`, but more options are coming soon, including custom CSS animation option.                                                                              |
```

Comment:
| `custom` | string | optional | It is you to define the purpose of the carousel. Currently, it only supports `banner`.